packages/core/src/interpreter.ts

```diff
@@ -320,6 +321,10 @@ export class Actor<
     return this;
   }
 
+  public getOutput() {
+    return this.logic.getOutput?.(this._state);
+  }
```

Comment:
IMHO it would be could to take another look at how we can standardize status thing, if it would be standard then perhaps we could read it without introducing yet another method to the logic interface. Output is already available within the return value of `getStatus` - it's just that there is no strong contract around the return type of this method so right now it's not that obvious that the logic implemented can do it.

https://github.com/statelyai/xstate/blob/0c2e56270c884abba5d5c200fbede11f071ad2a2/packages/core/src/interpreter.ts#L224

Reply:
~~Done~~ Started here: https://github.com/statelyai/xstate/pull/4199